docs/sources/mimir/release-notes/v2.8.md

```diff
@@ -13,10 +13,42 @@ The highlights that follow include the top features, enhancements, and bugfixes
+- **Experimental support for using Redis as cache** Mimir now can use Redis for caching results, chunks, index and metadata.
+- **Experimental support for fetching secret from Vault for TLS configuration**.
 
 ### Helm chart improvements
 
+The Grafana Mimir and Grafana Enterprise Metrics Helm chart is now released independently. See the [Grafana Mimir Helm chart documentation](/docs/helm-charts/mimir-distributed/latest/).
+
 ## Important changes
 
+In Grafana Mimir 2.8 we have removed the following previously deprecated or experimental configuration options:
+- The following metrics have been removed:
+  - `cortex_bucket_store_series_get_all_duration_seconds`
+  - `cortex_bucket_store_series_merge_duration_seconds`
+  - `cortex_ingester_tsdb_wal_replay_duration_seconds`
+
+The following configuration options are deprecated and will be removed in Grafana Mimir 2.10:
+
+- The CLI flag `-blocks-storage.tsdb.max-tsdb-opening-concurrency-on-startup` and its respective YAML configuration option `tsdb.max_tsdb_opening_concurrency_on_startup`.
+
+The following experimental options and features are now stable:
+
+- Use protobuf internal query result payload format by default
+
+
 ## Bug fixes
+
+- Querier: Streaming remote read will now continue to return multiple chunks per frame after the first frame. [PR 4423](https://github.com/grafana/mimir/pull/4423)
+- Store-gateway: the values for stage="processed" for the metrics cortex_bucket_store_series_data_touched and cortex_bucket_store_series_data_size_touched_bytes when using fine-grained chunks caching is now reporting the correct values of chunks held in memory. [PR 4449](https://github.com/grafana/mimir/pull/4449)
+- Compactor: fixed reporting a compaction error when compactor is correctly shut down while populating blocks. [PR 4580](https://github.com/grafana/mimir/pull/4580)
+- OTLP: Do not drop exemplars of the OTLP Monotonic Sum metric. [4063](https://github.com/grafana/mimir/pull/4063)
+- Packaging: flag /etc/default/mimir and /etc/sysconfig/mimir as config to prevent overwrite. [PR 4587](https://github.com/grafana/mimir/pull/4587)
+- Query-frontend: don't retry queries which error inside PromQL. [PR 4643](https://github.com/grafana/mimir/pull/4643)
+- Store-gateway & query-frontend: report more consistent statistics for fetched index bytes. [PR 4671](https://github.com/grafana/mimir/pull/4671)
+- Native histograms: fix how IsFloatHistogram determines if mimirpb.Histogram is a float histogram. [PR 4706](https://github.com/grafana/mimir/pull/4706)
+- Query-frontend: fix query sharding for native histograms. [PR 4666](https://github.com/grafana/mimir/pull/4666)
+- Ring status page: fixed the owned tokens percentage value displayed. [PR 4730](https://github.com/grafana/mimir/pull/4730)
+- Querier: fixed chunkIterator may return values after Seek beyond end of chunk. [PR 4450](https://github.com/grafana/mimir/pull/4450)
```

Comment:
Are we including all bug fixes here now? I think we should refer people to check the changelog, and only highlight most important bugfixes here.

Reply:
I will remove small bugfixes just leave out the important bugfixes.